cvat-ui/src/components/project-page/project-page.tsx

```diff
@@ -28,18 +28,16 @@ export default function ProjectPageComponent(): JSX.Element {
     const id = +useParams<ParamType>().id;
     const dispatch = useDispatch();
     const history = useHistory();
-    const projects = useSelector((state: CombinedState) => state.projects.current);
+    const projects = useSelector((state: CombinedState) => state.projects.current.map((project) => project.instance));
     const projectsFetching = useSelector((state: CombinedState) => state.projects.fetching);
     const deletes = useSelector((state: CombinedState) => state.projects.activities.deletes);
     const taskDeletes = useSelector((state: CombinedState) => state.tasks.activities.deletes);
     const tasksActiveInferences = useSelector((state: CombinedState) => state.models.inferences);
     const tasks = useSelector((state: CombinedState) => state.tasks.current);
-    const projectSubsets = useSelector((state: CombinedState) => {
-        const [project] = state.projects.current.filter((_project) => _project.id === id);
-        return project ? ([...new Set(project.tasks.map((task: any) => task.subset))] as string[]) : [];
-    });
 
     const [project] = projects.filter((_project) => _project.id === id);
+    const projectSubsets = [''];
```

Comment:
``No subset assigned``?

Reply:
It's a default value for the task without a subset.

Reply:
Yes, just on UI we could rename it as we wish for better UX, right?